pysb/simulator/scipyode.py

```diff
@@ -689,17 +679,3 @@ def _integrator_process(code_eqs, jac_eqs, num_species, num_odes, initials,
         trajectory[i:, :] = 'nan'
 
     return trajectory
```

Comment:
I see your PR removed the `SerialExecutor`, so I was curious whether restoring it for the `num_processors=1` case would make things faster (lack of extra process overhead). On my Mac laptop it seemed to make little to no difference in practice, so I think we can safely remove it.

Reply:
We could keep the SerialExecutor and use the map function with only one chunk instead of the submit function. That would get rid of some overhead from the process submit.

Reply:
I think the overhead is negligible since only one process is launched. Happy to leave as is.

Reply:
I always assumed the speedup from SerialExecutor was mostly from avoiding pickling all the inputs and outputs for each task. Was the overhead measured with sufficiently large input and/or output?